ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLDataFetchers.java

```diff
@@ -238,7 +239,8 @@ public GraphQLDataFetchers(final Set<Capability> supportedCapabilities) {
       final LogsQuery query =
           new LogsQuery.Builder().addresses(addrs).topics(transformedTopics).build();
 
-      final List<LogWithMetadata> logs = blockchainQuery.matchingLogs(fromBlock, toBlock, query);
+      final List<LogWithMetadata> logs =
+          blockchainQuery.matchingLogs(fromBlock, toBlock, query, new AtomicBoolean(true));
```

Comment:
We will want to add the zombie thread killer to graphql as well, so it may be worth a comment that the `AtomicBoolean(true)` prevents the query from self-terminating.